platforms/common/include/px4_platform_common/px4_work_queue/WorkQueueManager.hpp

```diff
@@ -64,7 +64,9 @@ static constexpr wq_config_t I2C2{"wq:I2C2", 1400, -10};
 static constexpr wq_config_t I2C3{"wq:I2C3", 1400, -11};
 static constexpr wq_config_t I2C4{"wq:I2C4", 1400, -12};
 
-static constexpr wq_config_t att_pos_ctrl{"wq:att_pos_ctrl", 6600, -13}; // PX4 att/pos controllers, highest priority after sensors
+// PX4 att/pos controllers, highest priority after sensors.
+// Stack size needed with ekf2 sideslip failure triggered was 6750
```

Comment:
I don't think we need to document the history in the actual header, especially if the git blame can take us back to the github PR.

Reply:
Fair.